### DISCUSSION TITLE
running setup in winpe checks hardware

### DISCUSSION THREAD

**Discussion by mchoff** [AveYo/MediaCreationTool.bat#173] [Q&A] [answered]
Hello! booting up from dvd made by created iso working great on my older pc (no checks yay).

 However, when booting up winpe (win10, wimboot) environment over pxe, and then mapping the drive to the created win11 iso folder to run setup stops with an unsupported hardware error. I didn't see anything related in the readme file, is that normal? or a way to proceed thru that way?

thanks<3

**Comment by AveYo** [comment, marked as answer]
You're not using boot.wim from media\sources - that's what gets used when booting usb / dvd.
You're booting via something else, so why would that environment bypass requirements?
But don't worry, I've got this scenario covered as well:
- simply launch auto.cmd from media, it will add the registry bypass keys and start setup

**Reply by mchoff** [reply]
Wow, thanks so much for going over that with me! That makes total sense, i'm very noobish and appreciate the pointers, you rock! 🙏💚😊

**Reply by AveYo** [reply]
Tx. 
It's not in readme, but it's noted in the `auto.cmd` near the top (main script lines 653 - 657)
```cmd
::# start sources\setup if under winpe (when booted from media) [Shift] + [F10]: c:\auto or d:\auto or e:\auto etc.
reg query "HKLM\Software\Microsoft\Windows NT\CurrentVersion\WinPE">nul 2>nul && (
 for %%s in (sCPU sRAM sSecureBoot sStorage sTPM) do reg add HKLM\SYSTEM\Setup\LabConfig /f /v Bypas%%sCheck /d 1 /t reg_dword
 start "WinPE" sources\setup.exe & exit /b 
) 
```

**Reply by mchoff** [reply]
my winpe pxe boot was using winpeshl.ini and a batch file to launch a command prompt for easy access to map a shared folder. I made a win11 pxe boot entry using the new win11 boot.wim, I added ini and batch file and it works! this was fun! thanks!

I know totally off topic but here's my ipxe entry in case anyone else wanted to see.
`
:win11aveyo

  kernel ${srv}/wimboot
  initrd ${srv}/Win11_21H2_AveYo/install.bat                    install.bat
  initrd ${srv}/Win11_21H2_AveYo/winpeshl.ini                 winpeshl.ini
  initrd ${srv}/Win11_21H2_AveYo/boot/bcd                    BCD
  initrd ${srv}/Win11_21H2_AveYo/boot/boot.sdi             boot.sdi
  initrd ${srv}/Win11_21H2_AveYo/sources/boot.wim       boot.wim
  boot
`
